### DISCUSSION TITLE
Error notifications

### DISCUSSION THREAD

**Discussion by redstormsju** [netalertx/NetAlertX#747] [General]
Hello all,

I’ve just noticed the following error notifications…can’t figure out the issue…anyone else experiencing similar?

"Error executing query (attempts: 10), query: UPDATE Parameters SET par_Value=\"[]\"\n WHERE par_ID=\"Front_Devices_Order\""
Or
"Error executing query (attempts: 10), query: INSERT INTO Parameters (par_ID, par_Value)\n VALUES (\"Front_Devices_Rows\",\n \"50\")"

**Comment by jokob-sk** [comment]
Hi @redstormsju ,

These can show up if the DB is locked. In general, these specific ones are only to set the default order of the tables, so this shouldn't cause any issues in the app.

**Reply by redstormsju** [reply]
Hi @jokob-sk How do I unlock the database?  This began occurring out of nowhere…meaning, I didn’t change any settings!!

**Comment by jokob-sk** [comment]
This should not be an issue. You could try increasing the delay between
scans to minimise the amount of DB operations. You could try to move the DB
on a faster drive if you have one, such as a ssd.

**Reply by redstormsju** [reply]
@jokob-sk How exactly do I increase the delay?  I can’t seem to find this option….and these errors keep my instance sort of freezing…not letting me navigate through settings! Sorry for the many questions…

**Reply by jokob-sk** [reply]
You can increase the delay by increasing the time between scans on any scanner. Make sure you use the same schedule for all enabled scanners:

![image](https://github.com/user-attachments/assets/eebfb886-bdc7-4093-9289-e14647107577)

**Reply by jokob-sk** [reply]
You can also unload unused plugins after disabling them to decrease the number of operations on the DB:

![image](https://github.com/user-attachments/assets/6211069e-71de-4f9f-abcc-fdcbb2c2e031)

**Comment by nathang21** [comment]
I've been getting this as well on a brand new install with the default docker config, seems like maybe something else is going on.
